browser/brave_content_browser_client.cc

```diff
@@ -255,6 +256,37 @@ void BraveContentBrowserClient::AdjustUtilityServiceProcessCommandLine(
 #endif
 }
 
+bool BraveContentBrowserClient::WillCreateURLLoaderFactory(
+    content::BrowserContext* browser_context,
+    content::RenderFrameHost* frame,
+    int render_process_id,
+    bool is_navigation,
+    bool is_download,
+    const url::Origin& request_initiator,
+    network::mojom::URLLoaderFactoryRequest* factory_request,
+    network::mojom::TrustedURLLoaderHeaderClientPtrInfo* header_client,
+    bool* bypass_redirect_checks) {
+  bool use_proxy = false;
+
+  use_proxy = ChromeContentBrowserClient::WillCreateURLLoaderFactory(
+      browser_context,
+        frame, render_process_id, is_navigation, is_download, request_initiator,
+        factory_request, header_client, bypass_redirect_checks);
+
+  // TODO(iefremov): Skip proxying for certain requests?
```

Comment:
is there anything to do for this right now? I guess this is the same as Chrome?

Reply:
Yeah, maybe we should do something with it in the future... For now, Chrome doesn't proxy some requests from internal WebUIs https://cs.chromium.org/chromium/src/extensions/browser/api/web_request/web_request_api.cc?q=web_request_api&sq=package:chromium&dr=C&l=650